index.html

```diff
@@ -78,7 +78,17 @@ <h3 class="code-line" data-line-start=2 data-line-end=3><a id="Notes_1"><a href=
 			</ul>
 		</details>
 		<p></p> 
-		<h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_relocate_Software_Engineers_from_Russia_9"></a><strong>GREAT</strong> companies that relocate Software Engineers from Russia, Belarus, Ukraine</h2>
+		<h2 class="code-line" data-line-start=9 data-line-end=10><a id="Companies_that_relocate_Software_Engineers_from_Russia_Ukraine_Belarus"></a><strong>GREAT</strong> companies that relocate IT specialists from Russia, Belarus, Ukraine</h2>
+		<p></p> 
+		<h4>Positions in high demand:</h4>
+		<ul>
+			<li>Frontend Software Engineer</li>
+			<li>Backend Software Engineer</li>
+			<li>iOS/Android Developer</li>
```

Comment:
I'm not sure how seo works, but would we have any search benefit if we fully split role like this?
iOS Developer
Android Developer

Reply:
I wanted to shorten the list as much as possible. From SEO perspective, my understanding is that there won't be a huge difference. And I could propose to merge Front and Back too then. As all other bullet points have a mix of keywords. Let me know!

Reply:
Ok, cool! Let's merge Frontent/Backend engineer. We can also add the Fullstack Software Engineer